### PR TITLE
chore: standardize release script

### PR DESCRIPTION
**Pull request by jye-sf**
Standardize the naming for our release scripts:

yarn release -> full service release
yarn release:version -> update the version
yarn release:publish -> publish the project

### CODE REVIEW

**Comment by jye-sf** [comment]
I've noticed we rarely update the version on this project, so I'm assuming we're publishing with the same version. Is that intentional, or should I be adding a version update into the process?

**Comment by jye-sf** [comment]
Assuming we want version updates, the general release process for our repos will be:

```
// Version update
yarn release:version

// github related tasks
git push origin <branch>
git push origin <tag>

// Publish
yarn release:publish
```

Github related tasks require unlocking and re-locking branch protections. Will look into having CircleCI handle that.

File: package.json [hunk 1]
```diff
@@ -15,7 +15,10 @@
     "lint": "eslint src",
     "build": "rollup -c",
     "test": "jest --coverage",
-    "release": "yarn build && npm publish --access public"
+    "release": "yarn build && yarn release:publish",
+    "release:version": "npm version",
+    "release:publish": "npm publish --access public --registry=https://registry.npmjs.org",
+    "release:publish:ci": "npm publish --access public --registry=https://registry.npmjs.org --dry-run"
```
Comment (jye-sf) [inline package.json:21]: Will remove `--dry-run` after review.

**Comment by jye-sf** [comment]
Will spend tomorrow refactoring all the other repos into `publish.js` with a CI check if the rest of the review looks good.

**Review by nolanlawson — APPROVED** [review]

**Pull request merged**